### DISCUSSION TITLE
AI生活化的應用—信義房屋案件實例

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#547] [General]
信義房屋目前引入了Microsoft Azure的服務，該服務內容主要是和ＡＩ相關。他們採用的服務主要是和資料分析和數據整合有關，藉由此服務讓消費者可以找到更符合自己需求的東西，另外也可以更方便的對已經有的資料做整合和管理。藉由這個例子，可以了解到人工智慧已經漸漸融入我們的生活當中了，
至於文章中提到的Microsoft Azure的服務，除了文章內提到的之外，還有像是人臉辨識、聊天機器人之類的服務。受限於篇幅的大小，之後有機會再為大家帶來更詳細的解說。
[https://tw.news.yahoo.com/%E4%BF%A1%E7%BE%A9%E6%88%BF%E5%B1%8B%E8%81%AF%E6%89%8B%E5%BE%AE%E8%BB%9F-%E6%89%93%E9%80%A0ai%E6%99%BA%E8%83%BD%E9%85%8D%E6%A1%88-072952353.html?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAADIfMpxVyLLieJl_EylleSByEJuC0DmgpPbGJhZ9PtrK1twQ7ZHCKFEdOylZb9lLWDY3_HsMT4pmnd-4DriemLQhAVyvaNHxKK46GvS7EnpBtWLfoglp0MgwSRSxON5Pgzwo_lOVMurtzvo5OQdbFW2m_9ZrYkll9UMHieh-GSS0](https://tw.news.yahoo.com/%E4%BF%A1%E7%BE%A9%E6%88%BF%E5%B1%8B%E8%81%AF%E6%89%8B%E5%BE%AE%E8%BB%9F-%E6%89%93%E9%80%A0ai%E6%99%BA%E8%83%BD%E9%85%8D%E6%A1%88-072952353.html?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAADIfMpxVyLLieJl_EylleSByEJuC0DmgpPbGJhZ9PtrK1twQ7ZHCKFEdOylZb9lLWDY3_HsMT4pmnd-4DriemLQhAVyvaNHxKK46GvS7EnpBtWLfoglp0MgwSRSxON5Pgzwo_lOVMurtzvo5OQdbFW2m_9ZrYkll9UMHieh-GSS0)

**Comment by treeann** [comment]
謝謝你的分享 讓我更了解AI生活化可以運用在什麼地方

**Reply by SamTU0222** [reply]
謝謝分享，人工智慧真的無所不在

**Comment by HuanYuuuuuu** [comment]
感覺AI會在生活中越來越普及了

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我更知道AI生活化的運用。

**Comment by Sazmafuji** [comment]
用AI尋找，用AI判斷，用AI決定......?

**Comment by PoooYen** [comment]
之後用AI來幫我們判斷CP值應該不是夢了吧..

**Comment by henrywen547** [comment]
謝謝分享!未來能期望減少消費者購屋與房仲業者服務的時間，進而增加效率！

**Comment by cordydeng** [comment]
之前上課有聽過microsoft azure服務，而這個應用對於買房的人和房仲業者也都有幫助，真的不得不佩服AI的力量!!

**Comment by gnsJhenJie** [comment]
有玩過Azure 的聊天機器人服務，建置不會很困難，大家也可以去玩玩看。

**Comment by sys511613** [comment]
AI在我們生活中應用的例子越來越多了
